lib/Swagger/src/Discussion/Models/HalProblem.php

```diff
@@ -63,11 +63,11 @@ class HalProblem implements ArrayAccess
       * @var string[]
       */
     protected static $swaggerTypes = array(
+        'instance' => 'string',
+        'type' => 'string',
         'status' => 'int',
-        'title' => 'string',
         'detail' => 'string',
-        'instance' => 'string',
-        'type' => 'string'
+        'title' => 'string'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)